### ISSUE TITLE
10/21: Anjali Adukia

### ISSUE THREAD

**Issue by shevajia** [uchicago-computation-workshop/Fall2021#6]
Comment below with questions or thoughts about the reading for this week's workshop.

Please make your comments by Wednesday 11:59 PM, and upvote at least five of your peers' comments on Thursday prior to the workshop. You need to use 'thumbs-up' for your reactions to count towards 'top comments,' but you can use other emojis on top of the thumbs up.

**Comment by Thiyaghessan** [comment]
Hi Professor Adukia,

Thank you for taking the time to share your work with us. I appreciate how succinctly the core arguments and methodology were presented. I was wondering if you had considered a few avenues for future work:

1.	Occupation as an additional variable
a.	I am wondering if it would be a valuable area for future research to look at the sorts of occupations individuals of differing racial and gender identities are shown holding in these books. In particular, if such representations are changing over time. Image recognition in this circumstance might be challenging but you could use text analysis to see if gender and racial minorities are associated with specific occupations. 
i.	For example, in Singapore books used in schools are mandated to include a diversity of racial groups. However, inequality perpetuates itself in the form of occupations members of racial minorities hold. For example, Indians and Malays will tend to be shown holding jobs that are considered less “prestigious” or associated with lower socioeconomic status in society

2.	Gender roles and actions undertaken
a.	I am also wondering if in the text analysis, we could see if the adjectives or verbs used to describe male/female presenting characters in these books are systematically different
i.	For example, are male characters associated with more aggressive actions than female characters?
b.	Additionally, if the prevalence of male-presenting characters engaging in activities not commonly associated with their gender identification increasing over time and vice versa
i.	For example, a male character engaging in “cooking” or “cleaning”, which are not the types of actions men have been traditionally associated with 
ii.	Of course I am wondering if this could be done via image analysis as you have done, but it also seems that image analysis would be much more challenging

**Comment by ZHE-ZHANG-0213** [comment]
Hi Professor Adukia! Thank you for sharing your work.
My question about your research is: I think people tend to describe individuals they know well, so whether your findings are related to the different levels of education of people of different colors in history. For the above reasons, there has always been a small number of black writers (in the past, the right of black people to receive an education was not fully reflected), which led to the characters in the book being flooded with light skin?
I look forward to seeing you on Thursday!

**Comment by MkramerPsych** [comment]
Dr. Adukia,

Thank you for sharing your research with us! I think this is an excellent application of machine learning methods to tackle an important issue in underrepresentation.

I am particularly curious as to what you believe the advantages are to using these computational methods over traditional methods in this type of work. It seems to me that one could come to the same conclusions about the underrepresentation of BIPOC characters in literature without the use of natural language processing or computer vision techniques. Beyond the ability to demonstrate these inequities in specific units, what are the other potential benefits of conducting these types of studies using machine learning? Are there questions that are specifically suited to these methods? Do the benefits of these techniques outweigh the potential costs (computation time, model interpretability, additional training time for researchers)?

**Comment by ChongyuFang** [comment]
Hi Prof Anajli Adukia, BIG thanks for attending this workshop! Your work is a lens through which we can take a serious look at the “black box” of education through children’s books in US schools and homes. Quantifying this is NOT easy. It was indeed excellent work. I look forward to the workshop tomorrow.

**Comment by Lynx-jr** [comment]
Hi Professor Adukia, thanks for coming to our workshop! Do you expect the inclusivity of authors/ books to vary in different languages and extent?

**Comment by hsinkengling** [comment]
Hi Prof. Adukia,
Thanks for coming to the workshop! I'm really excited to hear your talk.
There seem to be a lot of visual and linguistic markers for gender and race that help with automated content analysis. Do you think it's possible in the future to code for other, more subtle dimensions of representation? (eg. different personalities, emotions)

**Comment by lguo7** [comment]
Dear Prof. Anjali Adukia, 
Thank you very much for sharing your research. I am very interested in the artificial intelligence method you mentioned -- the way to turn images into data. Could you please introduce this method in detail? And how did you use artificial intelligence to scan images and text in so many children's books? How do you find this large amount of electronic materials?

**Comment by LFShan** [comment]
Hello Professor Adukia,
Thank you for the presentation. My question is that would Parents' inclination changes the type of book their children are able to acquire? If so, could it be the case that mainstream books would never be given to some children despite books' popularity among other children? Thank you.

**Comment by javad-e** [comment]
Thank you for presenting at our workshop!
I was wondering if you have any thoughts on books published in languages other than English? Particularly, do you know if children's books published in Northern Europe have better gender representation? Gender distribution in job market is slightly different in Scandinavian countries, so I was wondering whether we see a similar pattern in children's books.

**Comment by yutaili** [comment]
Hi Professor Adukia,

Thanks for sharing! I think your work is really interesting and insightful. In the paper, you've mentioned that more characters with darker skin color appear over time, but in the mainstream award-winning book these characters are still underrepresented. Do you think in a general sense that the effect of more minority characters appearing across all books and the effect of fewer minority characters in award-winning books will counterbalance each other? Or do you think the trend that minority group characters are underrepresented in children's books is not changing/still persist?

**Comment by a-bosko** [comment]
Dear Dr. Adukia,

Thank you for sharing your work! I am interested in the field of child psychology, so it was very interesting to learn about how books can shape a child's education and understandings about society. It was particularly interesting to learn about age differences in books, especially that adults appear more often than children in children's books. Do you believe that there should be more representation of children in books?

Thank you again, and I look forward to your presentation!

**Comment by Toushirow1** [comment]
Hi Professor Adukia, 
Your research on education inequality is compelling and insightful! Since education is an important manifestation of human capital accumulation. I look forward to seeing your thoughts about education inequality's impact on human capital accumulation and economic growth.

**Comment by LuZhang0128** [comment]
Hi Professor Adukia,
I previously read a [paper](https://www.jstor.org/stable/2657315) that "documents changes in racial images and examines the relationship between culture, gatekeeping, and conflict in society" by using "three sets of children's books." One of the sets is books that received the Caldecott Medal or were designated Caldecott Honor, which is also used in this research. However, previous researchers need to manually code the images. It's nice to see how image processing is making things easier now.

**Comment by taizeyu** [comment]
Dear Dr Adukia, I am so appreciated that you are able to make a presentation for us. I am so surprised that how can the researchers combine the computational method into the policy and social research.

**Comment by mikepackard415** [comment]
HI Dr Adukia,
Thank you for joining us at the CSS workshop. Very much looking forward to the presentation tomorrow.

**Comment by AlexBWilliamson** [comment]
Dear Dr. Adukia

Thank you for agreeing to presenting on this topic - I am looking forward to it!

**Comment by zoeyjiao1104** [comment]
Hi Professor Adukia, thank you for giving us the talk about your paper. This topic on race, gender and child's education is really interesting and meaningful. I am also very interested in your methodology of using Artificial Intelligence techniques to convert image into data. Looking forward to the talk!

**Comment by Emily-fyeh** [comment]
Hi Dr, Adukia, Thank you for sharing this working paper with us! Just curious about a small point of this paper: in the text analysis session, the words representing colors are counted and referred to as a proxy to race. However, colors in children's book may be used to describe other subjects, usually not exclusively for skin colors. Is it a common practice to    make this assumption?

**Comment by LynetteDang** [comment]
Hi Professor Adukia, thank you so much for sharing your work and I'm looking forward to tomorrow's workshop! Incorporating image processing as a toolbox into social science research was something I can't even imagine and am very excited about! In terms of image processing, I am wondering for such large image datasets (with RGB values of each pixel in 5,403 illustrated faces that you take as training set), what is your way of storing all the raw image data and processed data (I am assuming all pixel in a picture will be put down in a matrix)? And how did you optimize the runtime of your K-means clustering? Did you take advantage of SVD or some other techniques to reduce the dimensionality? Or did you take advantage of supercomputers to run your algorithms? Would you mind going into more details about AutoML Vision and the machine learning algorithms behind it? Also, since you have already came up with the idea of using image as data, have you ever thought about using audio as data with the current AI technologies that we have? I know that child audio books is also a thing and with the same model builder package you used (AutoML Vision), they also offer speech recognition/detection!

**Comment by wanxii** [comment]
Thanks for joining our workshop and look forward to your presentation!

**Comment by koichionogi** [comment]
Professor Anjali Adukia. Thank you so much for presenting your research. The methods and findings are really interesting and significant.  I assume there should be some unclassified images or characters, such as characters with androgyny features or hard to distinguish races, through Al analysis of books. I would like to know how you have dealt with and consider those characters. Would those characters be ignored due to irrelevance for representation comparison or would  they be classified into closest categories since those characters would still provide certain images to children? Thank you so much

**Comment by Qiuyu-Li** [comment]
Dr. Adukia,
Thanks for coming to the workshop! What people are taught in their childhood does have a great impact on the formation of their values, so I am really interested in the research.

**Comment by MengChenC** [comment]
Thank you for sharing your work. Look forward to your presentation!

**Comment by Coco-Jiachen-Yu** [comment]
Thank you for sharing with us your study, Dr. Adukia! What do you think are potential contributors to the phenomenon that White males are persistently overrepresented, despite a huge increase of female societal participation? Also, how do you think the results in this study can inform policy makers and educators?

**Comment by NaiyuJ** [comment]
Thank you for sharing your work. Look forward to your presentation!

**Comment by helyap** [comment]
Hi Professor Adukia, 

Thank you for presenting at the workshop tomorrow. Your application of text and image classification in this study is very exciting and I'm looking forward to seeing continued research work in this vein. As I was reading your paper, one of the questions that surfaced pertained to the background/profile of the authors of the books and if there are any patterns associated with your findings on the representativeness in the books. I am also curious about the awarding process and the decision-makers on these panels. Overall, similar to some of the other questions posted on the board, I am wondering what you think are contributors that affect representation in books and how they are valued.

**Comment by Qlei23** [comment]
Hi professor, thank you for coming to our workshop! I like your perspective of race and gender in children’s books and I find this research super exciting. I am looking forward to your presentation tomorrow!

**Comment by boyafu** [comment]
Thank you for sharing this fascinating research! I look forward to the presentation.

**Comment by BaotongZh** [comment]
Hello, Professor Adukia. 

Thank you for bring us this brilliant working paper. I look forward to the presentation. And also, in my opinion, the picture may have more information than text, therefore, implying more impact on children? The combination of images and text is a critical way to shape children's perspective.

**Comment by XinSu6** [comment]
Thank you so much for sharing you work! I am just wondering how you could do other stuff besides the presence of people in different media for others role? And on other fields?

Looking forward to your presentation!

**Comment by xxicheng** [comment]
Thank you for sharing your work with us!

**Comment by jinfei1125** [comment]
Hi Prof Adukia, thanks for joinning us and sharing your amazing work! I totally agree with your idea that representations of in texts and images of children's book. What do you think about the human biases in your research?

**Comment by mariadanielavadillo** [comment]
Hi Professor Adukia, thank you for coming to present, the paper was very insightful. I was wondering if there was a correlation between the authors' identities and the identity of the characters in the books they write (not just related to race, but also to gender, etc.). Furthermore, do you think it would be possible to apply a similar analysis to children's movies that could further corroborate your hypothesis? Since movies might also play a role in children's perception of race and representation.

**Comment by luckycindyyx** [comment]
Thank you so much for sharing such fascinating work with us! Looking forward to your lecture!

**Comment by Yutong0828** [comment]
Hi Professor Adukia, I really like the topic of your paper! I believe it's a very innovative way to study inequality in education process through analyzing the children's books. I was wondering now that you've detected such problems existing in children's books, like colored-skin people and women are underrepresented, do you have any thoughts to improve the status quo? Also, I would be very grateful if you could expand a bit on how you conduct text analysis. Thanks!

**Comment by Tanzi11** [comment]
I am looking forward to your presentation, Professor Adukia! Many thanks.

**Comment by jsoll1** [comment]
Your work seems super cool and I'm super excited for your presentation. I'm kind of curious about the wider implications with your results: you've rigorously verified using exciting new technology something that we already expected. How can we use this information to promote better behaviors?

**Comment by FranciscoRMendes** [comment]
I am excited to hear your talk!

**Comment by chentian418** [comment]
Thanks for your work, and look forward to your presentation!

**Comment by chuqingzhao** [comment]
Thank you for sharing your work. I look forward to your talk

**Comment by yhchou0904** [comment]
Thank you for sharing your work! The paper about children's books is very cool, and the idea could inspire us to think about what children are reading and what's the possible outcome of that.

**Comment by sdbaier** [comment]
Given that education increasingly occurs via comparably new forms of media (e.g., podcasts, videos, games), I would be curious to see how your findings translate into this context. I would expect the representation of race,  gender, and age to be less skewed in more novel forms of media, as they are more progressive/cutting-edge in nature.

**Comment by xzmerry** [comment]
Thanks for joining us, Prof. Adukia! It is really interesting that we could relate race and gender recognition/discrimination through the discussion of text of children's books. Looking forward to seeing you in the workshop!

**Comment by YileC928** [comment]
Looking forwards to learning more about your work in the workshop Prof. Adukia!

**Comment by qishenfu1** [comment]
Hi Professor, thank you very much!

**Comment by atowey-uchi** [comment]
Hi Professor, looking forward to learning more about your methods and insights!

**Comment by YijingZhang-98** [comment]
Hi Professor Adukia, thanks for sharing with us. Look forward to the speech!

**Comment by yierrr** [comment]
Thanks so much for the interesting paper. Looking forward to the presentation!

**Comment by TwoCentimetre** [comment]
Thanks for sharing.
